zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/Paragraph.java

```diff
@@ -601,10 +621,13 @@ public void setStatusToUserParagraph(Status status) {
     return config;
   }
 
+  // NOTE: function setConfig(...) will overwrite all configuration
+  // Merge configuration, you need to use function applyConfigSetting(...)
   public void setConfig(Map<String, Object> config) {
```

Comment:
Why not name it as `mergeConfig` it is is used for merge configuration ?

Reply:
Ok, I am going to change it.